sympy/physics/units/__init__.py

```diff
@@ -28,12 +27,13 @@
     expression expressed in another unit.
 
 """
+# isort:skip_file
```

Comment:
I'm not sure people will know what these are and I think it would follow convention more to put it at the top just below the `# _*_ coding: utf-8 _*_`. And note that it is related to pytest (or whatever it is).